docs/gathered-data.md

```diff
@@ -13,1039 +13,1984 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+None
+
+### Notes
+None
 
 
 ## ClusterOAuth
 
-fetches the cluster OAuth - the OAuth with name cluster.
+Collects the cluster OAuth - the OAuth with name cluster.
+
+### API Reference
+- https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/oauth.go#L50
+- https://docs.openshift.com/container-platform/4.3/rest_api/index.html#oauth-v1-config-openshift-io
+
+### Sample data
+- [docs/insights-archive-sample/config/oauth.json](./insights-archive-sample/config/oauth.json)
+
+### Location in archive
+| Version   | Path														|
+| --------- | --------------------------------------------------------	|
+| >= 4.2.0  | config/oauth.json 					                        |
+
+### Config ID
+`clusterconfig/oauths`
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/oauth.go#L50
-Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#oauth-v1-config-openshift-io
+### Released version
+- 4.2.0
 
-* Location in archive: config/oauth/
-* See: docs/insights-archive-sample/config/oauth
-* Id in config: clusterconfig/oauths
+### Backported versions
+None
+
+### Notes
+None
 
 
 ## ClusterOperatorPodsAndEvents
 
-collects information about pods
+Collects information about pods
 and events from namespaces of degraded cluster operators. The collected
 information includes:
 
 - Definitions for non-running (terminated, pending) Pods
 - Previous (if container was terminated) and current logs of all related pod containers
 - Namespace events
 
-  - Location of pod definitions: config/pod/{namespace}/{pod}.json
-  - Location of pod container current logs:
-    config/pod/{namespace}/logs/{pod}/{container}_current.log
-  - Location of pod container previous logs:
-    config/pod/{namespace}/logs/{pod}/{container}_previous.log
-  - Location of events in archive: events/
-  - Id in config: clusterconfig/operators_pods_and_events
-  - Spec config for CO resources since versions:
-  - 4.6.16+
-  - 4.7+
+### API Reference
+None
+
+### Sample data
+- [docs/insights-archive-sample/pod](./insights-archive-sample/pod)
+- [docs/insights-archive-sample/events](./insights-archive-sample/events)
+
+### Location in archive
+| Version   | Path																    |
+| --------- | ---------------------------------------------------------------------- |
+| >= 4.3.0  | config/pod/{namespace}/{pod}.json										|
+| >= 4.8.2  | config/pod/{namespace}/logs/{pod}/{container}_{current|previous}.log   |
+| >= 4.8.2  | events/																|
+
+> **Note**
+> This Gatherer generates 3 types of reports.
+
+### Config ID
+`clusterconfig/operators_pods_and_events`
+
+### Released version
+- 4.3.0
+
+### Backported versions
+- 4.6.35+
+- 4.7.11+
+
+### Notes
+This used to be part of `ClusterOperators`, it was split generating this gather at `4.8.2`.
+It used to be Specs for `ClusterOperators`, and it was initially included at `4.3` and backported to `4.2.10`.
```

Comment:
Am I supposed to understand this sentence? Beacuse I don't :)

Reply:
LOL... I'll try to put it in better words.

Reply:
I tried to improve this statement in the current docs.